data/project/ember/beta.md

```diff
@@ -5,14 +5,14 @@ filter:
   - /ember\./
   - /ember-template-compiler/
 repo: emberjs/ember.js
-initialVersion: 3.27.0 # Manually update, the prior version to the current beta. See https://libraries.io/npm/ember-source throughout
-lastRelease: 3.28.0-beta.6 # Manually update
-futureVersion: 3.28.0-beta.7 # Manually update
-finalVersion: 3.28.0 # Manually update
+initialVersion: 3.28.0 # Manually update, the prior version to the current beta. See https://libraries.io/npm/ember-source throughout
+lastRelease: 4.0.0-beta.3 # Manually update
+futureVersion: 4.0.0-beta.4 # Manually update
+finalVersion: 4.0.0 # Manually update
 channel: beta
-cycleEstimatedFinishDate: 2021-08-09 12:00:00 # Manually update, the expected date of the finalVersion release
-date: 2021-05-03 # Manually update, get date for `initialVersion`
-nextDate: 2021-07-19 12:00:00 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
+cycleEstimatedFinishDate: 2021-09-20 12:00:00 # Manually update, the expected date of the finalVersion release
+date: 2021-08-10 # Manually update, get date for `initialVersion`
+nextDate: 2021-08-30 12:00:00 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
```

Comment:
In a separate pull request, let's consider removing (1) estimates of future dates and (2) unused keys in these Markdown files.

When we (a core member or a contributor) blindly follow the instructions, we may end up showing contradictory information to the public, as shown below. The screenshot shows that `4.0.0-beta.4` will be released around August 30, but this pull request is made in September.

<img width="1050" alt="screenshot" src="https://user-images.githubusercontent.com/16869656/132285899-a6ec86f9-07b7-41e6-80cd-aaaa410b805a.png">